spec/yardstick/v2_client/test_centre_time_window_spec.rb

```diff
@@ -11,4 +11,36 @@
       expect(subject.upcoming_and_recent(token)).to eq([])
     end
   end
+
+  describe :proctors do
+    it 'create the right type of proctors' do
+      stub_path_response(:get, '/v2/sittings/83/proctors', 200, [{
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.